src/main/java/com/conveyal/datatools/editor/models/transit/TripPattern.java

```diff
@@ -48,57 +49,73 @@
 
     public List<TripPatternStop> patternStops = new ArrayList<TripPatternStop>();
 
-    public int getNumberOfTrips () {
-        FeedTx tx = VersionedDataStore.getFeedTx(this.feedId);
-        Collection<Trip> trips = tx.getTripsByPattern(this.id);
-        return trips == null ? 0 : trips.size();
+    // give the UI a little information about the content of this trip pattern
+    public transient Long numberOfTrips;
+    public transient Map<String, Long> tripCountByCalendar;
+
+
+    @JsonProperty("numberOfTrips")
```

Comment:
We _might_ be able to do this by putting and `@JsonInclude` annotation on the transient field.